app/Actions/Concerns/InteractsWithGitHub.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Actions\Concerns;
+
+use App\Configuration\LamboConfiguration;
+use App\LamboException;
+
+trait InteractsWithGitHub
+{
```

Comment:
@mattstauffer I have made all the methods protected since all of them are used internally by the class using the trait. I don't think they will ever be called by code using a class that implements the trait.

Does this make sense to you since I know you have opinions regarding visibility usage 😁

Reply:
Oh, that makes sense!